main/main.c

```diff
@@ -58,10 +58,10 @@
 #include "eps.h"
 #include "sband.h"
 #include "system.h"
+#include "dfgm.h"
 
 #include "sband_binary_tests.h"
 #include "deployablescontrol.h"
-#include "HL_system.h"
```

Comment:
Why is this removed?

Reply:
It was already included earlier on line 36 as <HL_system.h>